sstables/sstables.hh

```diff
@@ -811,6 +811,10 @@ public:
     // This will change sstable level only in memory.
     void set_sstable_level(uint32_t);
 
+    void set_new_run_identifier() {
```

Comment:
generate_new_identifier() would be a better name

Reply:
Agreed. will fix in v2. thanks